worlds/stardew_valley/regions.py

```diff
@@ -353,8 +354,81 @@ def inverted(self):
 ]
 
 
+# Attempt to implement remote loading of modded data if relevant (to avoid circular references and clutter)
+def loaded_mod_region_data(mod: str, data_type: str, world_options: StardewOptions):
+    loaded_mod_data = []
+    if mod in world_options[options.Mods]:
+        for mod_data in ModDataList:
+            if (mod_data.mod_name == mod) & (data_type == "Regions"):
+                for region in mod_data.regions:
+                    if not mod_data.regions[region]:
```

Comment:
Is this if really necessary? If the list of entrances is empty, it would just append an empty list, which seems fine to me? Unless I am missing something